core/api/v1/keytransparency.proto

```diff
@@ -310,23 +305,23 @@ service KeyTransparency {
   // Entries contain signed commitments to a profile, which is also returned.
   // TODO(gbelvin): Replace with GetUser
   rpc GetEntry(GetEntryRequest) returns (GetEntryResponse) {
-    option (google.api.http) = { get: "/v1/directories/{directory_id}/apps/{app_id}/users/{user_id}" };
+    option (google.api.http) = { get: "/v1/directories/{directory_id}/users/{user_id}" };
```

Comment:
Since {user_id} is the trailing resource ID, it can capture a value that contains forward slashes. I think your suggestion would put the string "users/alice@domain.com" into user_id instead of just "alice@domain.com".

I moved the /history path segment earlier for ListEntryHistory. I dropped :queue from QueueEntryUpdate, since just using the post HTTP verb is enough to distingush it.